.circleci/config.yml

```diff
@@ -432,14 +436,14 @@ jobs:
       # Cache the venv directory that contains dependencies
       - restore_cache:
           keys:
-            - cache-key-gpu-151-{{ checksum "setup.py"}}-{{ checksum "requirements-test.txt"}}
+            - cache-key-gpu-151-1-{{ checksum "setup.py"}}-{{ checksum "requirements-test.txt"}}
```

Comment:
given the setup.py change, this is not needed, is it ?

Reply:
right. I can remove it